### ISSUE TITLE
[Request] PLD: Replace Royal Authority with Atonement WITHOUT Royal Authority replacing its own combo

### ISSUE THREAD

**Issue by Diagonet** [MKhayle/XIVComboExpanded#321]
**Describe the feature you'd like**
Basically I would like the option to do this:
Fast Blade > Riot Blade > Royal Authority > Royal Authority (Atonement) > Royal Authority (Supplication)  > Royal Authority (Sepulchre)
It would be the ability to turn on "Royal Authority Atonement Feature" without needing to use the "Royal Authority Combo" feature.

**Describe alternatives you've considered**
Not sure, the alternative would be just keep pushing all buttons? haha

**Additional context**
I like using the basic 3 move combo from pally but pressing atonement 3 times is kinda annoying, that's all.

Btw, thanks a lot for this plugin, its awesome!

**Comment by MKhayle** [comment]
Oh sure, that's a very easy one. I'll take care of it today.

**Comment by MKhayle** [comment]
https://github.com/MKhayle/XIVComboExpanded/commit/2fafd6c7a0e4a4e77541ac1dc54cf639cc0a4af1
added there, I'll push an update later today after some testing on another change

**Comment by MKhayle** [comment]
Added in 1.5.1.5! https://github.com/MKhayle/XIVComboExpanded/commit/2aed5ee933fae9503cd4fa6d228f69fa627a7ce9

**Comment by Diagonet** [comment]
I was hoping it would be simple, thanks a lot! Just tested and it works perfectly

**Comment by kaedys** [comment]
@Diagonet FYI, once the UI rework in #323 is added, I've merged the new Atonement feature that was added for this and the prior one, and made it no longer require the Royal Authority Combo feature.  Same, actually, for the Divine Might feature, and most of the others.  It's still under the prior combo ID (the one that currently requires the Royal Authority Combo), though, so when that UI change lands, you'll need to go in and enable the Atonement feature.  As long as you don't also enable the Royal Authority Combo feature, you'll retain your desired behavior, where you still need to do the 1-2-3 for Royal Authority yourself, but Atonement and its combo chain replace Royal Authority.

Again, this is only once the new UI is released (and trust me, you'll know when that happens).
